### ISSUE TITLE
Structural over functional features

### ISSUE THREAD

**Issue by kamilest** [kamilest/brain-age-gnn#19]
The [Kaufmann et al.](https://doi.org/10.1038/s41593-019-0471-7) paper uses
* cortical thickness
* surface area  
* volume 
* (as well as Euler indices for quality control) 
for its [xgboost model](https://github.com/tobias-kaufmann/brainage) (with no functional timeseries data).

I have the cortical thickness data across the whole parcellation but for only the total surface area and volume (compared to surface areas/volumes per parcel as in the paper above).

Experimenting with fully connected layers (no graph layers) using functional only/structural only data gave r^2<0 for the timeseries models and r^2 up to 0.4 (r up to 0.6 compared to the above benchmark of ~0.93) for structural data models for the validation set.

[Parisot et al.](https://arxiv.org/abs/1703.03020) uses RFE with ridge classifiers, but not sure exactly for which features, will need to check.
